database/migrations/2017_12_20_173343_add_foreign_columns_to_users_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddForeignColumnsToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->integer('team_id')->nullable()->unsigned();
+            $table->integer('owner_team_id')->nullable()->unsigned();
+            $table->integer('role_id')->nullable()->unsigned();
+
+            $table->foreign('team_id')->references('id')->on('team');
+            $table->foreign('owner_team_id')->references('owner_id')->on('team');
+            $table->foreign('role_id')->references('id')->on('role');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropColumn(['team_id', 'owner_team_id', 'role_id']);
```

Comment:
You need to drop the foreign keys, also.

Do a migrate and rollback to check if everything works.